baremetal/metal3machine_manager_test.go

```diff
@@ -4146,15 +4146,15 @@ var _ = Describe("Metal3Machine manager", func() {
 		}),
 	)
 
-	type testCaseGetKubeadmControlPlaneName struct {
+	type testCaseGetControlPlaneName struct {
```

Comment:
Since we generalized it, let's make sure we don't use `Kcp` anywhere throughout the tests?

Reply:
I replaced all the KCP naming to make it generic